pulp_ansible/app/viewsets.py

```diff
@@ -224,6 +290,38 @@ class CollectionVersionSignatureViewSet(NoArtifactContentUploadViewSet):
     filterset_class = SignatureFilter
     queryset = CollectionVersionSignature.objects.all()
     serializer_class = CollectionVersionSignatureSerializer
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "create",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_model_or_obj_perms:ansible.change_collection",
+                    "has_model_or_obj_perms:ansible.view_collection",
+                    "has_model_or_obj_perms:ansible.change_ansiblerepository",
```

Comment:
CollectionSignatures are a content type in pulp_ansible, so I believe this should be the same as the first two conditions for CollectionVersion create.